### PR TITLE
Add Luxon supported options documentation to README

### PR DESCRIPTION
**Pull request by stockiNail**
This PR is adding documentation about which options can be set to configure Luxon.

As reported by PR #42 and issue #46, these could not be well-known.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,19 @@ Read more about jsDelivr versioning on their [website](http://www.jsdelivr.com/)
 
 ## Configuration
 
-Any date adapter options in the chart configuration will be passed through to Luxon's factory functions. See the [Luxon API docs](https://moment.github.io/luxon/api-docs/index.html#datetimefrommillis) for the supported options, such as `zone` and `locale`, that these functions accept.
+Any date adapter options in the chart configuration will be passed through to Luxon's factory functions.
+
+The following table describes the supported options, that these functions accept, which can be set to chart [time or time series scale](https://www.chartjs.org/docs/latest/axes/cartesian/time.html).
```
Comment (benmccann) [inline README.md]: The first comma is not necessary. I'm not quite sure what the second half of the sentence is trying to communicate
```suggestion
The following table describes the supported options that these functions accept.
```

File: README.md [hunk 2]
```diff
@@ -37,7 +37,19 @@ Read more about jsDelivr versioning on their [website](http://www.jsdelivr.com/)
...
+
+Namespace: `options.scales[scaleId].adapters.date`
+
+| Name | Type | Default | Description
+| ---- | ---- | ------- | -----------
+| `locale` | `string` | `undefined` | Set a locale to set on the resulting `DateTime` instance. By default, Luxon uses the system's locale. See [INTL locale](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#locale_identification_and_negotiation).
+| `numberingSystem` | `string` | `undefined` | Set the numbering system to set on the resulting `DateTime` instance. See [Luxon](https://moment.github.io/luxon/#/intl?id=numberingsystem) and [INTL numberingSystem](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#syntax) documentation.
+| `outputCalendar` | `string` | `undefined` | Set the output calendar to set on the resulting `DateTime` instance. See [Luxon](https://moment.github.io/luxon/#/calendars?id=output-calendars) and [INTL calendar](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#syntax) documentation.
```
Comment (benmccann) [inline README.md]:
```suggestion
| `outputCalendar` | `string` | `undefined` | Set the output calendar to set on the resulting `DateTime` instance. See [Luxon](https://moment.github.io/luxon/#/calendars?id=output-calendars) and [`Intl` calendar](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#syntax) documentation.
```

File: README.md [hunk 3]
```diff
@@ -37,7 +37,19 @@ Read more about jsDelivr versioning on their [website](http://www.jsdelivr.com/)
...
+| `setZone` | `boolean` | `undefined` | Override the zone with a zone specified in the string itself, if it specifies one. By default, Luxon uses `options.setZone=false`.
+| `zone` | `string`\|`Zone` | `undefined` | Set the zone to place the `DateTime` into. By default, Luxon uses `options.zone='local'`. See [Luxon](https://moment.github.io/luxon/#/zones?id=luxon-works-with-time-zones) and [INTL timeZone](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#syntax) documentation.
```
Comment (benmccann) [inline README.md]:
```suggestion
| `zone` | `string`\|`Zone` | `undefined` | Set the zone to place the `DateTime` into. By default, Luxon uses `options.zone='local'`. See [Luxon](https://moment.github.io/luxon/#/zones?id=luxon-works-with-time-zones) and [`Intl` timeZone](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#syntax) documentation.
```

**Comment by stockiNail** [comment]
@benmccann thank you very much for your review. My English is not so good! I have changed INTL to `Intl` also in the other options description where it was used.

**Review by benmccann — APPROVED** [review]

**Comment by benmccann** [comment]
thank you! looks like a nice change. I can't merge, so will need to wait for someone else for that

**Review by kurkle — APPROVED** [review]

**Pull request merged**